adapters/repos/db/repo.go

```diff
@@ -78,6 +78,7 @@ type Config struct {
 	MaxImportGoroutinesFactor float64
 	FlushIdleAfter            int
 	TrackVectorDimensions     bool
+	WantDimensionsReindex     bool
```

Comment:
Maybe it would be better if the config variable name would match the env variable name: `ReindexVectorDimensionsAtStartup`?